src/app/pipes/daysLeftCountedPipe/days-left-counted.pipe.ts

```diff
@@ -0,0 +1,18 @@
+import { ActionItem } from 'src/app/services/tasksService/tasks.service';
+import { Pipe, PipeTransform } from '@angular/core';
+import { DaysLeftToDeadlineService } from 'src/app/services/daysLeftToDeadlineService/days-left-to-deadline.service';
+
+@Pipe({
+  name: 'daysLeftCounted'
+})
+export class DaysLeftCountedPipe implements PipeTransform {
+  constructor(private daysCountService: DaysLeftToDeadlineService) {}
+  transform(value: ActionItem, currentDate: Date = new Date()): number | undefined {
```

Comment:
**FIX:** If you pass just property instead of the action item object then this pipe would be more generic and could be used for the other cases.